.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is with.
+
+**Operator context**
+How did you install / run the operator (e.g. Helm, Docker image, operator-sdk, ...)?
+
+```
+# IMPORANT: Please make sure to anonymize any PII like IP addresses and usernames!
+```
+
+**Operating System context (regarding local development, remove section if not applicable)**
+
+**Kubernetes context**
+Kubernetes Server version:
+
+**To Reproduce**
+Steps to reproduce the behavior
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
Shouldn't these come first, directly after the bug description?

Reply:
Yup, now they do!